### ISSUE TITLE
Possible to decouple $axios from $auth?

### ISSUE THREAD

**Issue by nathanchase** [Teranode/nuxt-module-alternatives#55]
Since the Nuxt team is pushing for using [ohmyfetch](https://github.com/unjs/ohmyfetch) (aka [useFetch](https://v3.nuxtjs.org/guide/features/data-fetching#usefetch), [useAsyncData](https://v3.nuxtjs.org/guide/features/data-fetching#useasyncdata)) over axios because "[Axios is already expired for modern web era](https://github.com/nuxt/framework/discussions/4514#discussioncomment-2617603)" - is there a possibility we could rip out the auth-module's reliance on axios and just use `$fetch`?

I'm currently using $axios solely to log in/log out, and then for all subsequent calls just pulling the auth token and using $fetch - but it would be great to just eliminate the need for $axios entirely.

**Comment by Teranode** [comment]
I don't think I'll be going that far ahead with the auth module, though the idea had crossed my mind since I was able to make `@nuxtjs-alt/http` a drop-in replacement for `@nuxtjs-alt/axios` while being a layer over ohmyfetch, but as it stands, the nuxt team is working on an auth module themselves. I only did this as to help people migrate to nuxt3 while they wait for the official auth module to finish developing. With that said it wouldn't make sense for me to put in all that effort only for it to be replaced when their module is released.
